backend/src/graphql/model/Transaction.ts

```diff
@@ -42,7 +42,9 @@ export class Transaction {
     this.creationDate = transaction.creationDate
     this.linkedUser = linkedUser
     this.linkedTransactionId = transaction.linkedTransactionId
-    this.transactionLinkId = transaction.transactionLinkId
+    this.transactionLinkId = transaction.contribution
```

Comment:
if you combine it please name it `linkId`. If you do not combine it have both fields